content/blogs/2024-04-05-getting-started-with-kestra.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "Getting Started with Kestra"
+description: "If you're new to Kestra, this post will introduce you step by step to building your first workflows with Python and configuring notifications"
+date: 2024-04-05T10:00:00
+category: Solutions
+author:
+  name: Will Russell
+  image: "wrussell"
+image: /blogs/2024-04-05-getting-started-with-kestra.png
+---
+
+Kestra is an event driven data orchestration platform that’s highly flexible and easy to use. This guide is going to go through the basics and get you started building your own pipeline!
+
+Originating as a platform for data orchestration, Kestra finds itself well-equipped to manage all types of pipelines with its highly flexible interface and a huge range of plugins. Through this blog post, I’m going to show you how to get setup with Kestra and set up a simple workflow to run a Python script every hour and send the result as a Discord notification.
+
+## Installation
+
+Kestra is open source meaning anyone can run it on their machine for free. There is a cloud version coming soon for those who want to start making flows straight away, however for now we'll set it up locally. To get it setup, you’ll need to make sure you have Docker installed on your machine and run the following command to start up your instance!
```

Comment:
```suggestion
Kestra is open source meaning anyone can run it on their machine for free. To get it setup, you’ll need to make sure you have Docker installed on your machine and run the following command to start up your instance!
```

given Beta is coming in August by the earliest, maybe let's not mention just yet 👍